### ISSUE TITLE
Postprocess freehand - edit points

### ISSUE THREAD

**Issue by lein1013** [excalidraw/excalidraw#3916]
It seems that with the merge of   [Improve free-hand drawing](3500) feature the old feature where we could edit the points of a freehand drawing got lost (replaced by improved free-hand drawing).

Since I don't do very good freehand drawings the capability to edit the freehand lines after drawing was essential to create a good excalidraw (adding, removing points, smoothing the free-hand spline)

Is there any option to set to enable this again?

**Comment by dwelle** [comment]
There were two reasons for changing the way how free-hand drawing works:

1. Low-quality results of the previous algorithm. The lines ended up too jagged and the simplification algorithm usually resulted in lines you didn't draw in the first place.
2. The free-draw deep down was just a line tool, implementation-wise.

The point (2) is why migrating old drawings to the new free-draw was as simple as changing the `type` from `draw` to `line` to keep them looking the same after the migration.

In other words, if what you're after is a need to create "pixel" perfect shapes, you don't need the old free-draw tool, just use the line tool. You can edit/add the points the same way (we also plan to improve this workflow). Chances are you'll end up with a cleaner workflow (you won't need to delete/reposition as many points).

![excal_line_tool_as_freedraw](https://user-images.githubusercontent.com/5153846/129445182-54e4993f-fec8-4e2c-b339-0858312cb920.gif)

**Comment by dwelle** [comment]
But to answer your original questions: post-processing the new freedraw shapes would be harder: a small change in position of one point usually has a drastic effect on the simplification/rendering algorithm.

**Comment by steveruizok** [comment]
Perhaps we could add that type of drawing support with the line tool while holding option? I forget exactly how it worked but IIRC the code is still in the product.
